src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -111,16 +112,25 @@ def prepare_summary_data(
     duration: int,
     organization_id: int,
 ):
-    # build 'Today's Event Count vs. 14 day average'. we need 15 days of data for this
-    COMPARISON_PERIOD = 14
-    comparison_offset = ONE_DAY * COMPARISON_PERIOD + 1
     organization = Organization.objects.get(id=organization_id)
+    ctx = build_summary_data(
+        timestamp=timestamp, duration=duration, organization=organization, daily=True
+    )
+    return ctx
+    # XXX(CEO): in a follow up PR this will delivery the daily summary
+
+
+def build_summary_data(
```

Comment:
Might be easier to read if you look at the whole file, but I moved building the context up into it's own function that returns the context. This makes it easier to test for the next PR where I won't be returning anything in `prepare_summary_data`, but rather calling a `deliver_summary` function that sends the notifications.